pkg/virt-handler/isolation/isolation.go

```diff
@@ -53,9 +54,9 @@ import (
 type PodIsolationDetector interface {
 	// Detect takes a vm, looks up a socket based the VM and detects pid, cgroups and namespaces of the owner of that socket.
 	// It returns an IsolationResult containing all isolation information
-	Detect(vm *v1.VirtualMachineInstance) (*IsolationResult, error)
+	Detect(vm *v1.VirtualMachineInstance) (IsolationResult, error)
```

Comment:
This is because we transform IsolationResult into an interface (that is needed to properly mock it in unit test context). As for a separate commit, I will take a look.